tests/unit/aiplatform/test_training_jobs.py

```diff
@@ -1069,6 +1090,18 @@ def test_run_call_pipeline_service_create_distributed_training(
 
         assert job.state == gca_pipeline_state.PipelineState.PIPELINE_STATE_SUCCEEDED
 
+    @pytest.mark.usefixtures("get_training_job_mock")
+    def test_get_training_job(self, get_training_job_mock):
+        aiplatform.init(project=_TEST_PROJECT)
+        training_jobs.CustomTrainingJob.get(training_job_name=_TEST_NAME)
+        get_training_job_mock.assert_called_once_with(name=_TEST_NAME)
+
+    @pytest.mark.usefixtures("get_training_job_mock")
+    def test_get_training_job_with_id_only(self, get_training_job_mock):
+        aiplatform.init(project=_TEST_PROJECT, location=_TEST_LOCATION)
+        training_jobs.CustomTrainingJob.get(training_job_name=_TEST_ID)
+        get_training_job_mock.assert_called_once_with(name=_TEST_NAME)
+
```

Comment:
Didn't add the other units tests from `Dataset.init` because they are already covered by those tests.